components/setting_picture.tsx

```diff
@@ -15,35 +14,48 @@ import OverlayTrigger from 'components/overlay_trigger';
 import Tooltip from 'components/tooltip';
 import LoadingWrapper from 'components/widgets/loading/loading_wrapper';
 
-export default class SettingPicture extends Component {
+type Props = {
+    clientError?: ReactNode;
+    serverError?: ReactNode;
+    src?: string | null;
+    defaultImageSrc?: string;
+    file?: File | null;
+    loadingPicture?: boolean;
+    submitActive?: boolean;
+    onRemove?: () => void;
+    onSetDefault?: (() => Promise<void>) | null;
+    onSubmit?: (() => void) | null;
+    title?: string;
+    onFileChange: (e: React.ChangeEvent<HTMLInputElement>) => void;
+    updateSection: (e: React.MouseEvent<HTMLButtonElement>) => void;
+    imageContext?: string;
+    maxFileSize?: number;
+    helpText?: any;
```

Comment:
I was trying not to use any here, but it looked like the test file may have some typing issues. I can try and do that file as well to fix it, but didn't want to expand the scope of the ticket without asking first.

Reply:
Thanks for contributing. Feel free to fix test file to avoid the usage of any here

Reply:
Done, thanks :+1:

Reply:
thank you for migrating test file too. i can confirm there was no separate ticket for that 😄